docs/src/main/sphinx/release/release-472.md

```diff
@@ -0,0 +1,96 @@
+# Release 472 (26 Feb 2025)
+
+## General
+
+* Color console output for improved logs readability. ({issue}`25090`)
+* Suggest configuration for unused properties in log output. ({issue}`25090`)
```

Comment:
fyi @wendigo

Reply:
Just `Suggest configuration for unused properties`

Reply:
well.. where is that suggestion though?

Reply:
Also .. do we need to document that at all? I kinda think not but we could ..

Reply:
Part of the error message